anydex/wallet/bitcoinlib/bitcoinlib_wallets.py

```diff
@@ -1,289 +1,289 @@
-import os
```

Comment:
Why is this part of the merge request? Has anything been changed?

Reply:
Maybe it's CRLF to LF?

Reply:
I meant the removal and re-addition of the entire python file.

Reply:
Yes, that's how CRLF to LF gets shown in a commit, it just replaces all the lines